### DISCUSSION TITLE
Meeting: April 16, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#103] [Announcements]
## Weekly meetings happen on Tuesdays at 9am - 10am Pacific

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by handrews** [comment]
* Update from @hudlow and @mkistler on query strings
* @handrews has some thoughts building on @darrelmiller 's idea (as shared last week by @mkistler) of two schemas for parameters: one for each parameter, and a separate one for relationships among them

**Comment by hudlow** [comment]
@handrews points out that it might make more sense for @hudlow and @mkistler to work solving the parameter deserialization ambiguity in 3.x (likely 3.3 or 3.4) and use whatever we build there as the foundation for more advanced features in Moonwalk.

**Comment by handrews** [comment]
In answer to the question of why parameter interdependencies that came up, I found this talk (which I have not actually watched):  https://www.youtube.com/watch?v=fyOlqao2iXM

I'll also add it to the relevant discussion.

**Comment by handrews** [comment]
Actually, @mkistler already included a link to that presentation in the intro comment on #100.
